packages/venia-ui/lib/components/ProductSort/productSort.js

```diff
@@ -76,6 +76,7 @@ const ProductSort = props => {
                 priority={'low'}
                 classes={{ root_lowPriority: classes.sortButton }}
                 onMouseDown={handleSortClick}
+                onTouchStart={handleSortClick}
```

Comment:
I believe this is the first and only time we've done something like this in the app. You should add a comment above the return explaining that this is temporary and will be replaced by `onClick={handleSortClick}` once the search bar stops shifting layout on blur.

Reply:
I created a ticket and called out this specific line to be changed but I can add something in code too so the pattern isn't repeated elsewhere

Reply:
Nevermind! 🚨 

Instead of moving these handlers "forward" (`click` -> `mousedown`), in https://github.com/magento/pwa-studio/pull/2685/commits/7900de96f68dc90559ef720935424aef7d91adb0 I instead moved the `useDropdown` handler "back" (`mousedown` -> `click`).

I think this is preferable because it has fewer side effects throughout the rest of the codebase (like the lines commented here).

I'd like to verify with @jimbo any potential negative side effects of this change but clicking around in the five places we use `useDropdown` right now, it works great.

I should also note that I had to set the `capture` option of `useDropdown`'s click handler to `true` otherwise it was possible to get the search and my account dropdowns open at the same time (the original bug). Unfortunately I can't _really_ explain why that would be the case. It seems like it should work even if `capture` was `false`.